robotoff/app/api.py

```diff
@@ -948,7 +962,8 @@ def get_questions_resource_on_get(
         avoid_voted_on=_get_skip_voted_on(auth, device_id),
     )
 
-    insights = list(get_insights_(limit=count))
+    offset: int = (page - 1) * count
```

Comment:
here we could add a condition, such that if `order_by=='random'` then `offset=0` don't know if it improves perf

Reply:
yes good idea